README.md

```diff
@@ -42,28 +44,94 @@ Parameters
     Using more than 8 is wasteful, better put that memory
     in width instead or use 'basic' or 'conservative' algorithms instead.
 -   **Algorithm**: There are several algorithms available:
-    -   *basic* straightforward implementation of the Count-Min Sketch as described on
-        [Wikipedia](https://en.wikipedia.org/wiki/Count%E2%80%93min_sketch) with a cell size of 4 bytes.
-    -   *conservative* variant of the algorithm using a simple optimization which is also described on Wikipedia. Also
-        uses 4 bytes per cell.
-    -   *logcounter8* is a probabilistic counter with a value that fits into 1 byte. Values of this counter are usually
-        off by +/- 30%. This can be used for rough estimations for extremely large sets.
-    -   *logcounter1024* is a probabilistic counter with a value that fits into 2 bytes. Values of this counter are usually
-        off by +/- 2%.
-    -   *logcons1024* is a conservative-update upgrade of 'logcounter1024' which offers substantially better precision of
-        the structure.
+    -   *conservative* Count-min Sketch as described on as described on [Wikipedia](https://en.wikipedia.org/wiki/Count%E2%80%93min_sketch).
+        Uses 4 bytes per cell, storing values up to 2^32 - 1
+    -   *log1024* is a conservative-update probabilistic counter with a value that fits into 2 bytes. Values of this counter
+        up to 2048 are precise and larger values are off by +/- 2%. The maximum value is larger than 2^71
+    -   *log8* is a conservative-update probabilistic counter that fits into just 1 byte. Values of this counter
+        up to 8 are precise and larger values can be off by +/- 30%. The maximum value is larger than 2^33        
 
 Memory
 ------
 To calculate memory footprint:
     ( width * depth * cell_size ) + HLL size
 
 Cell size is
-   - 4B for basic / conservative algorithm
-   - 1B for logcounter8
-   - 2B for logcounter1024 / logcons1024
-HLL size is 32 KB
+   - 4B for conservative algorithm
+   - 2B for log1024
+   - 1B for log8
+   
+HLL size is 64 KB
 
 Example:
-    width 2^26 (67 108 864), depth 4, logcons1024 (2B) has 2^(26 + 2 + 1) + 32 768 = 536.9 MB
+    width 2^25 (33 554 432), depth 8, logcons1024 (2B) has 2^(25 + 3 + 1) + 64 KB = 536.9 MB
     Can be pickled to disk with this exact size.
+
+Performance
+-----------
+### Testing on Wikipedia set
+We have counted unigrams and bigrams in English Wikipedia dataset to evaluate the counters.
+In each case, we have counted the entire data set into all bounter structures:
+
+```python
+    with smart_open('title_tokens.txt.gz') as wiki:
```

Comment:
You should explain what the input file is. Probably not something directly downloaded from Wikipedia.

Reply:
@piskvorky Can you provide some insight about where the file 'title_tokens.txt.gz' comes from?

Reply:
@menshikh-iv recently added a script to gensim that generates this file.